### ISSUE TITLE
Unable to pass boundingBoxes prop

### ISSUE THREAD

**Issue by RashmiRamdas** [codeceptjs/codeceptjs-resemblehelper#116]
Hi Team,

Could you please add support for boundingBoxes similar to boundingBox/ignoredBox (https://github.com/codeceptjs/codeceptjs-resemblehelper/issues/12)

```js
await visual.seeVisualDiffForElement(selector, `test.png`, {
            prepareBaseImage: false,
            tolerance: 1,
            needsSameDimension: false,
            boundingBoxes: null
        });
```

**Comment by sharath2106** [comment]
We have released the latest version ([1.9.7](https://www.npmjs.com/package/codeceptjs-resemblehelper)) of the library yesterday. I would ask to check against the latest version and let us know.

**Comment by sharath2106** [comment]
Closed as there's no response from the user.
